### DISCUSSION TITLE
v0.6.1 released

### DISCUSSION THREAD

**Discussion by abartesaghi** [nextpyp/.github#47] [Announcements]
This release contains new features, improvements and bug fixes based on user feedback. See the [changelog](https://nextpyp.app/files/pyp/0.6.1/docs/changelog.html) for details.

To upgrade, follow the instructions described in the [docs](https://nextpyp.app/files/pyp/0.6.1/docs/install/install-web.html#upgrading-to-a-new-version).

**Comment by JuhaHuiskonen** [comment]
Is it possible to install v0.6.1 without root?

**Reply by abartesaghi** [reply]
Yes, it should be possible using the regular `install` script.
